frontend/apps/filemanager/filemanager.lua

```diff
@@ -1479,11 +1492,10 @@ function FileManager:showOpenWithDialog(file)
 end
 
 function FileManager:openFile(file, provider, doc_caller_callback, aux_caller_callback)
-    if not provider then -- check associated
-        local provider_key = DocumentRegistry:getAssociatedProviderKey(file)
-        provider = provider_key and DocumentRegistry:getProviderFromKey(provider_key)
+    if provider == nil then
```

Comment:
This is for clarity?

Reply:
If you mean `nil`, I think it's just a matter of taste, now I tend to use `not` for boolean values only. Not strong opinion.